Content.Shared/_RMC14/Sentry/SentrySpikesComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Content.Shared.Damage;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._RMC14.Sentry;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[Access(typeof(SentrySystem))]
+public sealed partial class SentrySpikesComponent : Component
+{
+    [DataField(required: true), AutoNetworkedField]
+    public DamageSpecifier SpikeDamage = default!;
```

Comment:
You can just make this new() so that an admin doesn't get pranked when using VV to add the component
```suggestion
    public DamageSpecifier SpikeDamage = new();
```